rbi/core/kernel.rbi

```diff
@@ -3162,4 +3162,20 @@ module Kernel
   # ```
   sig {returns(T.self_type)}
   def itself(); end
+
+  private def initialize_clone(*args); end
+
+  private def initialize_copy(orig); end
```

Comment:
This is already defined earlier in the file?

https://github.com/sorbet/sorbet/blob/5a623a107dc081d1143508dbe8353220090a126d/rbi/core/kernel.rbi#L491-L492

Reply:
Ah thanks nice catch. I suppose I should also alphabetize these.